src/OrchardCore.Modules/OrchardCore.Layers/Drivers/LayerMetadataWelder.cs

```diff
@@ -25,6 +25,11 @@ public LayerMetadataWelder(ILayerService layerService, IStringLocalizer<LayerMet
         protected override void BuildPrefix(ContentItem model, string htmlFieldPrefix)
         {
             Prefix = "LayerMetadata";
+
+            if (!String.IsNullOrEmpty(htmlFieldPrefix))
+            {
+                Prefix = htmlFieldPrefix + "." + Prefix;
+            }
```

Comment:
Same thing can be achieved without modifying base class and utilizing base class. No need to duplicate logic of base class here.

```
 base.BuildPrefix(model, htmlFieldPrefix);
  if(string.IsNullOrWhiteSpace(htmlFieldPrefix))
      Prefix = "LayerMetadata";
```